docs/rfc/README.md

```diff
@@ -0,0 +1,5 @@
+---
+order: 1
+parent:
+  order: false
+---
```

Comment:
I'm guessing the table of contents will go here. Do we also want to add a few lines that reiterate what you've said in this ADR in describing what are the conditions of an RFC and the expected process

Reply:
Yes, I left this blank for now till we discussed it. I'll fill it in before merging.

Reply:
I have updated the README with some more detail, PTAL.